UI/DevScreens.h

```diff
@@ -120,7 +120,7 @@ class SystemInfoScreen : public TabbedUIDialogScreenWithGameBackground {
 class AddressPromptScreen : public PopupScreen {
 public:
 	AddressPromptScreen(std::string_view title) : PopupScreen(title, "OK", "Cancel"), addrView_(NULL), addr_(0) {
-		memset(buttons_, 0, sizeof(buttons_));
+		memset(buttons_, 0, sizeof(*buttons_));
```

Comment:
this is also wrong, revert it.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18971/commits/39af23aefa8517e17b9ade1d4c065c47d50901e7